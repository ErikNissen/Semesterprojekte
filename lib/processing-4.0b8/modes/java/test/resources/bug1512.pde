version https://git-lfs.github.com/spec/v1
oid sha256:fab0940941dc35eff40aa554f60e1c9cfd260149933a1109362f2c4ef3cb44ca
size 17
