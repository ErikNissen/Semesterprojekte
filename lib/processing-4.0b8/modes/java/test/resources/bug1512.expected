version https://git-lfs.github.com/spec/v1
oid sha256:e9bf35ccc51f433efc0d9e0590cee70c6017dc045f60054609fc12008fdd701f
size 711
